app/controllers/event.js

```diff
@@ -163,7 +168,11 @@ module.exports = {
         notification.content = `Event ${deleteEvent.eventName} is deleted!`
         notification.tag = 'Deleted'
         notificationHelper.addToNotificationForAll(req, res, notification, next)
-        return res.status(HttpStatus.OK).json({ deleteEvent: deleteEvent, message: 'Deleted the event' })
+        activityTracker.addToRedis(req, res, next, collectionTypes.EVENT, event._id)
```

Comment:
@Rupeshiya 
I am little bit curious about tracking delete actions. I assume that once the event (or any deleted entity) is deleted, how does the `event._id` would play a role? How are you going to know which event (or any deleted entity) as `_id` becomes a dangling reference?

Reply:
@kmehant For better understanding go ahead fetch locally and run tests. 🎉

Reply:
Thanks for pointing out. ✌️